ffv1.md

```diff
@@ -27,7 +27,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 `Container`: Format that encapsulates `Frames` (see (#frame)) and (when required) a `Configuration Record` into a bitstream.
 
-`Sample`: The smallest addressable representation of a color component or a luma component in a `Frame`. Examples of `Sample` are Luma, Blue Chrominance, Red Chrominance, Transparency, Red, Green, and Blue.
+`Sample`: The smallest addressable representation of a color component or a luma component in a `Frame`. Examples of `Sample` are Luma (Y), Cb Chroma, Cr Chroma, Transparency, Red, Green, and Blue.
```

Comment:
I suggest "Luma (Y), Blue-difference Chroma (Cb), Red-difference Chroma (Cr)" (wording used in https://en.wikipedia.org/wiki/YCbCr) for more coherency in the usage of full name vs acronym.

Reply:
updated